ext/opcache/jit/zend_jit.c

```diff
@@ -4287,7 +4287,7 @@ static int ZEND_FASTCALL zend_runtime_jit(void)
 			/* perform real JIT for this function */
 			zend_real_jit_func(op_array, NULL, NULL);
 		} zend_catch {
-			do_bailout = 0;
+			do_bailout = true;
```

Comment:
could you set to false above for consistency ? your changes looks good otherwise.

Reply:
ah if that s the case, makes sens then.